.github/workflows/deploy-create-artifact.yml

```diff
@@ -0,0 +1,62 @@
+name: 'Deploy / Create Artifact'
+
+on:
+  workflow_call:
+    inputs:
+      retention-days:
+        required: false
+        type: number
+        default: 1
+
+env:
+  ARTIFACT_NAME: 'coblocks-plugin'
+  ARTIFACT_PATH: 'coblocks'
+  ARTIFACT_ZIP: 'coblocks.zip'
+
+# First build our plugin, then create and store the artifact.
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        # with:
```

Comment:
Commented out code?

Reply:
It was commented out, until we are ready to run the release. Fixed now.